packages-exp/functions-types-exp/index.d.ts

```diff
@@ -20,16 +20,16 @@ import { FirebaseError } from '@firebase/util';
 /**
  * An HttpsCallableResult wraps a single result from a function call.
  */
-export interface HttpsCallableResult {
-  readonly data: any;
+export interface HttpsCallableResult<ResponseData = any> {
+  readonly data: ResponseData;
 }
 
 /**
  * An HttpsCallable is a reference to a "callable" http trigger in
  * Google Cloud Functions.
  */
-export interface HttpsCallable {
-  (data?: {} | null): Promise<HttpsCallableResult>;
+export interface HttpsCallable<RequestParams = any, ResponseData = any> {
```

Comment:
Should the generics default to `unknown` instead?

Reply:
Before when I tried it I had some problems with the types not getting inferred in usage like if I did
```
const callTest = httpsCallable<string, string>(functions, "callTest");
const result = await callTest();
```
both `callTest` and `result` would be `any`. But maybe I just neglected to cast the return value in `httpsCallable` and I did that now and it seems to work.